src/shared/install.c

```diff
@@ -955,6 +958,8 @@ static int install_info_add(
 
         i = install_info_find(c, name);
         if (i) {
+                i->auxiliary = MIN(i->auxiliary, auxiliary);
```

Comment:
using MIN() on two bools is a bit weird, no?

```c
i->auxiliary = i->auxiliary && auxiliary;
```
would do the same and be less surprising, no? would even be shorter...